assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -549,3 +549,7 @@ che.server.secure_exposer.jwtproxy.memory_limit=128mb
 # Maximum size of the json processing pool
 # in case if pool size would be exceeded message execution will be rejected
 che.core.jsonrpc.processor_max_pool_size=100
+
+# CORS filter settings
+che.cors.allowed_origins=*
```

Comment:
Please consider adding a few words about the purpose of these values or add link to documentation of original CORS filter documentation.